### DISCUSSION TITLE
Having problem with user authentication

### DISCUSSION THREAD

**Discussion by Tomi-Philips** [supabase/supabase#27962] [Questions]
`
![code 2](https://github.com/user-attachments/assets/2026e620-f207-4e06-a7bf-43241f8d63b5)
![code](https://github.com/user-attachments/assets/7d455af9-541b-4e46-a3cb-d177d6c43c95)
![web pagew](https://github.com/user-attachments/assets/e81f62c2-9792-4e14-b47e-2c35d3bc7209)
`

**Comment by GaryAustin1** [comment]
Not a very well formatted post with much info.
Not being able to send email could be lots of things.
If you are using the provided SMTP connection, that is limited to 3 emails.  You should use your own SMTP provider.
Otherwise more info from the auth logs, or your own code with logging of the info from the response might be useful.
